src/Ledger/Transaction.lagda

```diff
@@ -51,8 +51,8 @@ Ingredients of the transaction body introduced in the Conway era are the followi
 \begin{itemize}
   \item \txvote, the list of votes for goverance actions;
   \item \txprop, the list of governance proposals;
-  \item \txdonation, the treasury donation amount;
-  \item \curTreasury, the current value of the treasury.
+  \item \txdonation, amount of \Coin to donate to treasury, e.g., to return money to the treasury after a governance action;
+  \item \curTreasury, the current value of the treasury. This field serves as a precondition to executing Plutus scripts;
```

Comment:
I think this should be a bit clearer. This may make you think that you cannot run them if this field isn't present, but that's not the case. It's just that they cannot see the value if it wasn't supplied.

```suggestion
  \item \curTreasury, the current value of the treasury. This field serves as a precondition to executing Plutus scripts that need to access the value of the treasury;
```